app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -226,14 +257,31 @@ class DatasetInfoTabView extends React.PureComponent<DatasetInfoTabProps> {
       Store.getState().temporaryConfiguration.controlMode === ControlModeEnum.VIEW;
 
     const zoomLevel = calculateZoomLevel(this.props.flycam, this.props.dataset);
-
+    const extentInVoxel = getDatasetExtentInVoxel(this.props.dataset);
+    const extent = getDatasetExtentInLength(this.props.dataset);
+    const datsetExtents = [
+      `${extentInVoxel.width} x ${extentInVoxel.height} x ${extentInVoxel.depth}`,
+      formatExtentWithLength(extent),
```

Comment:
Slight refactoring potential: How about `formatExtentWithLength` also accepts a second parameter which is a function which formats a value. So, you would call `formatExtentWithLength(extent, formatNumberToLength)`. And in the line above, you could write `formatExtentWithLength(extent, x => x)` since the pure voxel count is not formatted.
That way, these two lines would unified a bit and the ` x ` logic only exists once.

Reply:
Independent of that: I wouldn't define these two extents in an array and access them individually. Instead, maybe just inline them into the `<td />`s each?